content/tutorials/guides/how-to-collect-product-variations.md

```diff
@@ -0,0 +1,277 @@
+<a href="/tutorials/homepage.html" class="back-button">
+   <button>
+      <img src="/img/icons/icon--arrow-back.svg" style="margin-right: 10px;">
+      All guided tutorials
+   </button>
+</a>
+
+# How to collect product variations
+
+<table class="tag-container">
+    <tr>
+        <td>Use case:</td>
+        <td>
+            <div class="tag-not-selectable">
+                <div class="tag-color tag-color-light-blue"></div>
+                <div class="tag-label">App Workflow</div>
+            </div>
+        </td>
+    </tr>
+    <tr>
+        <td>PIM Features:</td>
+        <td class="td-features">
+            <div class="tag-not-selectable">
+                <div class="tag-color tag-color-purple"></div>
+                <div class="tag-label">Product Models</div>
+            </div>
+            <div class="tag-not-selectable">
+                <div class="tag-color tag-color-light-green"></div>
+                <div class="tag-label">Variant Products</div>
+            </div>
+        </td>
+    </tr>
+</table>
+
+<div class="endpoint-container">
```

Comment:
API endpoints aren't correct, see the Notion page.
https://api.akeneo.com/api-reference.html#get_products_uuid

https://api.akeneo.com/api-reference.html#get_product_models

https://api.akeneo.com/api-reference.html#get_families__family_code__variants__code__

Reply:
products
product models
family variants